metasv/breakdancer_reader.py

```diff
@@ -71,7 +71,7 @@ def parse_header_line(self, header_line):
             self.header_dict["software"] = header_line.split()[1]
         elif header_line.startswith("#Command:"):
             self.header_dict["command"] = header_line.split()[1:]
-        elif not (header_line.startswith("#Library") or header_line.startswith("#Chr1")):
+        elif not (header_line.startswith("#Library") or header_line.startswith("#Chr1")): #marghoob, what?
```

Comment:
It's actually trying to parse the header into a dictionary. BreakDancer has columns named chr1, pos1, orientiation1, chr2, pos2, orientation2 for the SVs. It isn't really chromosome 1. :)